app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -1245,6 +1245,11 @@ public void onTabsClose(ArrayList<Session> aTabs) {
     @Override
     public void onTabsReceived(@NotNull List<TabData> aTabs) {
         WindowWidget targetWindow = mFocusedWindow;
+
+        if (!aTabs.isEmpty()) {
+            targetWindow.getSession().setActive(false);
+        }
+
```

Comment:
I think if we are fullscreen this will just set the fullscreen Session to inactive?

Reply:
yes, fullscreen needs to be checked. You can move the setActive false to before this line https://github.com/MozillaReality/FirefoxReality/blob/2667446b82846e279c01628c5383109cfc4221de/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java#L1265

Reply:
Fixed